commit 29a43c20faa06100ade61fc24a5ee3bddfa3021a
Author: Lukas Tribus <lukas@ltri.eu>
Date:   Tue Aug 14 11:39:35 2018 +0200

    DOC: dns: explain set server ... fqdn requires resolver
    
    Abhishek Gupta reported on discourse that set server [...] fqdn always
    fails. Further investigation showed that this requires the internal
    DNS resolver to be configured. Add this requirement to the docs.
    
    Must be backported to 1.8.
    
    (cherry picked from commit c5dd5a500a237780eb9ab6e7069949cb19b6ff7d)
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/doc/management.txt b/doc/management.txt
index 68a17c25..46e7fd07 100644
--- a/doc/management.txt
+++ b/doc/management.txt
@@ -1675,7 +1675,8 @@ set server <backend>/<server> weight <weight>[%]
   equivalent of the "set weight" command below.
 
 set server <backend>/<server> fqdn <FQDN>
-  Change a server's FQDN to the value passed in argument.
+  Change a server's FQDN to the value passed in argument. This requires the
+  internal run-time DNS resolver to be configured and enabled for this server.
 
 set severity-output [ none | number | string ]
   Change the severity output format of the stats socket connected to for the
